packages/esm-service-queues-app/src/routes.json

```diff
@@ -22,50 +22,61 @@
       },
       "online": true,
       "offline": true
-    }, 
+    },
+    {
+      "name": "queue-table-by-status-menu-dashboard-link",
+      "component": "queueTableByStatusMenu",
+      "meta": {
+        "name": "service-queues",
+        "slot": "service-queues-dashboard-slot",
```

Comment:
I'm forgetting, why do we need to have the slot listed in the meta here?

Reply:
It's just the way the home page app extensions work.  I think it basically serves to connect the link component with the dashboard component that it renders.